protocol/messenger_backup.go

```diff
@@ -26,6 +26,12 @@ var backupTickerInterval = 120 * time.Second
 // backups
 var backupIntervalSeconds uint64 = 28800
 
+type CommunitySet struct {
+	Joined  []*communities.Community
+	Left    []*communities.Community
+	Deleted []*communities.Community
```

Comment:
I don't think there is a semantic difference between `Left` and `Deleted` communities. I found the originating PR: https://github.com/status-im/status-go/pull/2639/ and the issue: https://github.com/status-im/status-mobile/issues/13198, which states: "Communities you left are not backed up/synced".

Newly introduced `LeftCommunities` seems to be a subset of `DeletedCommunities`:

LEFT:
```
query := communitiesBaseQuery + ` WHERE NOT c.Joined AND NOT c.spectated AND r.state != ?`
```

DELETED:
```
query := communitiesBaseQuery + ` WHERE NOT c.Joined AND (r.community_id IS NULL or r.state != ?)`
```